lib/menu.rb

```diff
@@ -0,0 +1,20 @@
+class Menu
+
+  attr_reader :dish
+
+  DEFAULT_MENU = { margherita: 8, romana: 10, four_cheese: 12, seafood: 14, garlic_bread: 5 }
+
+  def initialize (dish=DEFAULT_MENU)
+    @dish = dish.sort
+  end
+
+  def read
+    @dish.to_h
```

Comment:
I'm wondering why you are doing this here.
You may want to read this to understand better how hashed and arrays differ, especially around ordering: http://www.rubyinside.com/how-to/ruby-sort-hash